patches/src/main/kotlin/app/revanced/patches/instagram/story/flipping/DisableAutoStoryFlippingPatch.kt

```diff
@@ -0,0 +1,17 @@
+package app.revanced.patches.instagram.story.flipping
+
+import app.revanced.patcher.patch.bytecodePatch
+import app.revanced.util.returnEarly
+
+@Suppress("unused")
+val disableAutoStoryFlippingPatch = bytecodePatch(
+    name = "Disable auto story flipping",
```

Comment:
```suggestion
    name = "Disable story auto flipping",
```

And rename patch file, and apidump